src/screens/Shell.tsx

```diff
@@ -1,21 +1,18 @@
 import { useEffect } from "react";
```

Comment:
I'm not a fan of the name of this component.  It's pretty vague.  Shell for what?  What's the purpose?

Reply:
It is the Shell of the entire Portal. Everything except the login page is rendered inside the shell. It could be called 'Wrapper' I guess? But that seems vague as well.

Reply:
Maybe `AppShell`?